packages/odyssey-react/src/components/Button/Button.module.scss

```diff
@@ -173,22 +178,22 @@
 .wideLayout {
   display: block;
   width: 100%;
-  margin-block: 0;
```

Comment:
We may not want to make these themable. Adding margins here would prevent the button from displaying correctly - making it not full width.

I'm realizing now that "wide" was not a sufficient name when we designed this.

Reply:
@edburyenegren-okta I'll remove